tests/bluetooth/host/mock_host/src/iso.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <string.h>
+#include <stdio.h>
+#include <errno.h>
+#include <zephyr/sys/atomic.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/slist.h>
+#include <zephyr/sys/byteorder.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/direction.h>
+#include <zephyr/bluetooth/iso.h>
+#include <host/hci_core.h>
+
+
```

Comment:
can you add a comment here ? 
```c
/* This file doesn't contain mocks, but rather fakes of the necessary parts of
 * subsys/bluetooth/host/iso.c. The API implementations that are copied here
 * should be kept in sync with the original.
 */
```

Reply:
also see https://www.martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs
and
http://xunitpatterns.com/Test%20Double.html

Reply:
Thanks for sharing these pages, they were really helpful